cmd:col is no longer provided by util-linux-misc
since 2.40, it was disabled for musl libc in
https://github.com/util-linux/util-linux/commit/8886d84e25a457702b45194d69a47313f76dc6bc

--- a/autoconf/targets.mak
+++ b/autoconf/targets.mak
@@ -298,7 +298,7 @@
 # Format a manpage into plain text
 define MANIFY
 	@$(ECHO) "  MAN  " $(1) -\> $(2)
-	$(V)man ./$(1) | col -b > $(2)
+	$(V)mandoc -Tascii ./$(1) > $(2)
 endef
 
 # Rule to build a Windows resource object from the source RC file
